var/spack/repos/builtin/packages/py-devito/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDevito(PythonPackage):
+    """Devito is a Python package to implement optimized stencil computation.
+
+    (e.g., finite differences, image processing, machine learning) from high-level
+    symbolic problem definitions. Devito builds on SymPy and employs automated code
+    generation and just-in-time compilation to execute optimized computational kernels
+    on several computer platforms, including CPUs, GPUs, and clusters thereof.
+    """
+
+    homepage = "https://www.devitoproject.org/"
+    pypi     = "devito/devito-4.6.2.tar.gz"
+
+    version('4.6.2', sha256='39c2210a192ad69953b4f8d93440ffd72b07d739c4fe2290e2b182adfb7e143f')
+
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('matplotlib', default=False, description='Enable matplolib support')
+    variant('pandas', default=False, description='Enable pandas support')
+
+    depends_on('py-pip@9.0.1:', type='build')
+    depends_on('py-versioneer', type='build')
```

Comment:
Where do you see a versioneer dep?

Reply:
Here: https://github.com/devitocodes/devito/blob/144c4bb09facc7402e2684c9d2d230c5a530dee8/setup.py#L1

Could this also be resolved by the local `versioneer.py` inside the project?